### PR TITLE
Judgement Bird buffs

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Attack speed decreased from 10 seconds to 8 seconds
Attack now roots Jbird in place.

Range increased from 10 to 12.
Attacking Jbird with pale (his weakness) now increases damage by 5 and range by 1 (resets on next ding)

## Why It's Good For The Game
They must fucking suffer.

## Changelog
:cl:
balance: Jbird buffs
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm [hunk 1]
```diff
@@ -93,3 +105,8 @@
 		datum_reference.qliphoth_change(-1)
 	return
 
+/mob/living/simple_animal/hostile/abnormality/judgement_bird/attackby(obj/item/I, mob/living/user, params)
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm:108]: Exec won't activate this. Include same code in Bullet_Act so I can dual wield this man into a map nuke please.

**Review by dsmith328 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm [hunk 2]
```diff
@@ -93,3 +105,8 @@
...
+	..()
+	if(I.damtype == PALE_DAMAGE)
+		judgement_range += 1
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm:111]: Jesus christ no. Never.
This is beyond fucked up.

File: code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm [hunk 3]
```diff
@@ -55,6 +56,11 @@
 /mob/living/simple_animal/hostile/abnormality/judgement_bird/AttackingTarget(atom/attacked_target)
 	return OpenFire()
 
+/mob/living/simple_animal/hostile/abnormality/judgement_bird/Move()
+	if(judgement_active == TRUE)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm:60]:
```suggestion
	if(judgement_active)
```

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Increasing damage and range with each pale attack is not needed at all.

**Pull request closed without merging**